packages/data/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types"
+	},
+	"references": [
+		{ "path": "../stan" },
+	],
+	"include": [
+		"src/atomic-store/*",
```

Comment:
@sirreal I tried to type that file which seems very easy, but for some reason, it's not working as expected. I could use help 😬